content/issues/2022-10-13-cluster-upgrade.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "System upgrade scheduled"
+date: 2022-10-13T19:00:00+00:00
+resolved: false
+resolvedWhen: 
+section: issue
+severity: notice
+---
+A system upgrade is scheduled for Wed Oct 13 19:00 UTC.
```

Comment:
We can be more specific about what can be expected. What about something like this:
```suggestion
A system upgrade is scheduled for Wed Oct 13 19:00 UTC.
We do not expect any significant disruption.
The started jobs might be executed with a delay
and if you think Packit does not react to your events,
try to retrigger the action via
[comment command](https://packit.dev/docs/guide/#how-to-re-trigger-packit-actions-in-your-pull-request).
```